# This msg file specifies all arrangement states related to
# the object specified

int32 obj_idx
uniform_object_rearrangement/ArrState[] invalid_arr_states